src/utils/colorUtils.ts

```diff
@@ -83,7 +83,7 @@ export const getColor = (
 
 	const type = getColoringType(props);
 	if (type) {
-		const color = get(props.theme, `colors.${type}`) as Theme['colors']['key'];
+		const color = get(props.theme.colors, type) as Theme['colors']['key'];
```

Comment:
You missed this intentionally?
```suggestion
		const color = props.theme.colors?.[type];
```

Reply:
That's only valid if `type` is a simple string @thgreasi, if it includes `.` or `[` it could do magic, eg `a.b.c` or `a[0].b`. I've been trying to work through these and simplify all the ones where it looks unintentional to support that magic but I haven't convinced myself for all of them yet

Reply:
Indeed you are right :+1: